scriptmodules/supplementary/skyscraper.sh

```diff
@@ -457,7 +407,7 @@ function _gui_advanced_skyscraper() {
 
 function gui_skyscraper() {
     if pgrep "emulationstatio" >/dev/null; then
-        printMsgs "dialog" "This scraper must not be run while Emulation Station is running or the scraped data will be overwritten. \n\nPlease quit from Emulation Station, and run RetroPie-Setup from the terminal"
+        printMsgs "dialog" "This scraper must not be run while Emulationstation is running or the scraped data will be overwritten.\n\nPlease quit Emulationstation and run RetroPie-Setup from the terminal:\n\n sudo \$HOME/RetroPie-Setup/retropie_setup.sh"
```

Comment:
actually - it's technically `EmulationStation`

Reply:
That clears up the confusion :). Will fix everywhere.